components/Starknet/modules/tools/pages/limits_and_triggers.adoc

```diff
@@ -24,7 +26,7 @@ The gas limit for Starknet blocks is therefore inherited from the gas limit for
 
 |Max transaction size (Cairo steps)|The maximum number of computational steps, measured in Cairo steps, that a transaction can contain when processed on the Starknet network.
 This limit is important for ensuring the efficient execution of transactions and preventing potential congestion.
-| 3,000,000 | 3,000,000 | 3,000,000
+| 4,000,000 | 4,000,000 | 4,000,000
```

Comment:
@odednaor Yesterday I was told that the new limits only apply to Sepolia and Mainnet. Are you sure this applies to Goerli as well?